src/webgl/interaction.js

```diff
@@ -132,18 +132,24 @@ p5.prototype.orbitControl = function(
   }
   this._renderer.prevTouches = this.touches;
 
-  // flags for interaction
-  let executeZoomProcess = false;
-  let executeRotateProcess = false;
-  let executeMoveProcess = false;
+  // The idea of using damping is based on the following website. thank you.
+  // https://github.com/freshfork/p5.EasyCam/blob/master/p5.easycam.js
```

Comment:
We might want to consider using a link to a commit hash (and maybe a relevant line number?) in case the easycam repo changes any branch or file names. Maybe this? https://github.com/freshfork/p5.EasyCam/blob/9782964680f6a5c4c9bee825c475d9f2021d5134/p5.easycam.js#L1124

Reply:
Certainly, it would be easier to understand if put a link directly to the place. I got it!
In the case of comments, is it okay if the number of characters is large?

Reply:
It seems to be fine!